docs/spec/spec.md

```diff
@@ -0,0 +1,380 @@
+# Specification: Ballerina WebSubHub Library
+
+_Owners_: @shafreenAnfar @chamil321 @ayeshLK    
+_Reviewers_: @shafreenAnfar    
+_Created_: 2021/11/23  
+_Updated_: 2021/11/23  
+_Issue_: [#786](https://github.com/ballerina-platform/ballerina-standard-library/issues/786)
+
+# Introduction
+
+This is the specification for WebSubHub standard library which is used to implement WebSub compliant `hub` services 
+and `publisher` clients using [Ballerina programming language](https://ballerina.io/), which is an open-source programming language for the 
+cloud that makes it easier to use, combine, and create network services. 
+
+# Contents  
+1. [Overview](#1-overview)  
+2. [Hub](#2-hub)
+   * 2.1. [Hub Listener](#21-hub-listener)
+   * 2.2. [Hub Service](#22-hub-service)
+     * 2.2.1. [Service Annotation](#221-service-annotation)
+3. [Hub Client](#3-hub-client)
+4. [Publisher Client](#4-publisher-client)
+
+## 1. Overview
+
+[WebSub](https://www.w3.org/TR/websub/) is a real-time content delivery protocol over HTTP(s) and it is a simple 
+specification which evolved from [PubSubHubbub](https://github.com/pubsubhubbub/PubSubHubbub).
+
+WebSub specification describes three main roles: 
+- Publisher: An implementation that advertises a topic and hub URL on one or more resource URLs.
+- Subscriber: An implementation that discovers the `hub` and topic URL given a resource URL, subscribes to updates at 
+the `hub`, and accepts content distribution requests from the `hub`.
+- Hub: An implementation that handles subscription requests and distributes the content to subscribers when the 
+corresponding topic URL has been updated.
+
+`WebSubHub` is an API framework which is derived from the WebSub specification which could be used by developers to 
```

Comment:
framework should be library.

Reply:
Fixed with [e9ec5bccc217575bffcb58e22d9f2bb4edfcef94](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/378/commits/e9ec5bccc217575bffcb58e22d9f2bb4edfcef94)